src/modules/core/components/DropdownMenu/DropdownMenuItem/DropdownMenuItem.tsx

```diff
@@ -5,10 +5,11 @@ import styles from './DropdownMenuItem.css';
 const displayName = 'DropdownMenuItem';
 
 const DropdownMenuItem = ({
+  className,
   children,
   ...props
 }: HTMLAttributes<HTMLLIElement>) => (
-  <li className={styles.main} {...props} role="menuitem">
+  <li className={`${styles.main} ${className}`} {...props} role="menuitem">
```

Comment:
There's not theme here, so here, the correct course of action is to overwrite the class. _(so revert the change again)_

If you need to inherit styles for your overwrite, you can use the `composes:` declaration available from CSS Modules